.github/workflows/build-master.yml

```diff
@@ -0,0 +1,57 @@
+name: Build
+
+on:
+  repository_dispatch:
+    types: [stdlib-publish-snapshot]
+  workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
This should be change to `main`, check all places

Reply:
refer : https://github.com/ballerina-platform/module-ballerina-random/tree/main/.github/workflows

Reply:
I will change

Reply:
Changed. Commit id is 7384fe53d9cec3b3fb1954da256cdf322c4b605a